Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -139,10 +252,10 @@ def intersection(lst1, lst2):
     }
 
 
-# change the keys of a dictionary according to a conversion map
-# trans_map - { 'OldKey': 'NewKey', ...}
-# camelize - change all keys from snake_case to CamelCase
 def replace_keys(src, trans_map=FIELD_NAMES_MAP, camelize=True):
+    """Function to change the keys of a dictionary according to a conversion map."""
+    # trans_map - { 'OldKey': 'NewKey', ...}
+    # camelize - change all keys from snake_case to CamelCase
```

Comment:
I suggest we remove these lines

Reply:
Hi @guykeller (cc: @barchen1 ), Made the code change to fix the issue and pushed the new version of the Tenable_io.py file to the branch in my fork.

Reply:
Done
Update 02.23.2021 1034 HRS EST:
Removed lines from code